src/picongpu/include/plugins/PluginController.hpp

```diff
@@ -35,6 +35,7 @@
 #include "plugins/PositionsParticles.hpp"
 #include "plugins/BinEnergyParticles.hpp"
 #include "plugins/ChargeConservation.hpp"
+#include "plugins/particleDetector/ParticleDetector.hpp"
```

Comment:
this is wrong.

as with radiation, this include should *only* be made if `HDF5` is available. otherwise you will crash all users that do not use HDF5, e.g., all people benchmarking the core code.